indico/web/client/js/jquery/widgets/jinja/permissions_widget.js

```diff
@@ -271,18 +271,25 @@ import Palette from 'indico/utils/palette';
       ];
       this.$permissionsWidgetList.append(this._renderItem(anonymous));
 
-      const managersTitle =
-        this.options.objectType === 'event'
-          ? $T.gettext('Category Managers')
-          : $T.gettext('Event Managers');
-      const categoryManagers = [
+      var managersTitle;
+      if (this.options.objectType === 'category') {
+        managersTitle = $T.gettext('Category Managers');
+      } else {
+        managersTitle = $T.gettext('Parent Category Managers');
+      }
```

Comment:
That logic here isn't correct:

- event -> category managers
- category -> parent category managers
- anything else -> event managers